pkg/crypto/tinkcrypto/unwrap_support.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package tinkcrypto
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/golang/protobuf/proto"
+	hybrid "github.com/google/tink/go/hybrid/subtle"
+	"github.com/google/tink/go/keyset"
+	tinkpb "github.com/google/tink/go/proto/tink_go_proto"
+
+	ecdhespb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/ecdhes_aead_go_proto"
+)
+
+func extractPrivKey(kh *keyset.Handle) (*hybrid.ECPrivateKey, error) {
+	buf := new(bytes.Buffer)
+	w := &privKeyWriter{w: buf}
+	nAEAD := &noopAEAD{}
+
+	if kh == nil {
+		return nil, fmt.Errorf("extractPrivKey: kh is nil")
```

Comment:
Similar comment as the one I made in the unwrap code about the name of the function being in the error message here

Reply:
same response, `extractPriveKey` is the function name, it's good practice to start errors with the function name to help troubleshooting (especially in prod where logs don't show much info about the source of an error)